README.md

```diff
@@ -1,77 +1,98 @@
+should be opt-in (if possible), and properly reflect data collection with the `permissions` manifest key.
+
+### 7. Limit use of external APIs
 
-## 7. Limit use of external APIs
+External APIs, even open source, may secretly do malicious actions, as returning malicious data, logging your IP
+address and more. If you have to use your API or another one that is not provided by a well-known entity (like Spotify
+API), please add the `ext_api` permission to your manifest.
 
-External APIs, even open source, may secretely do malicious actions, as returning malicious data, logging your IP address
-and more. If you have to use your API or another one that is not provided by a well-known entity (like Spotify API),
-please add the `ext_api` permission to your manifest.
+**Note**: If your plugin requires the use of an external custom-built backend, Powercord is ready to hook you up with
+free hosting and domain (`yourplugin.powercord.dev`). [More about that here](https://github.com/powercord-community/suggestions#about-backends).
 
-## 8. Do not break other plugins
+### 8. Avoid breaking other plugins
 
-Your plugin should be able to run in a way that it'll **not** affect any other plugin installed. We internally made
-sure everything will run smoothly even if multiple plugins are patching the same component.
+Your plugin should be able to run, no matter what plugins the user installed. We internally have some protection to help
+preventing conflicts, but we can't do everything for you. If your plugin happens to be incompatible with another, try
+discussing with the plugin developer.
 
-## 9. Prefer contributing instead of duplicating
+### 9. Prefer contributing instead of duplicating
 
 If you want to make a plugin that already exists, prefer contributing to the already existing plugin than having another
 plugin with the same purpose.
 
-## 10. Do **NOT** perform manual HTTP requests to Discord or the Gateway
+### 10. Only perform manual HTTP requests to Discord if there are no other solutions
 
-Discord internally has everything you'll need to retrieve data without directly calling the API or connecting to the
-Gateway. Performing such actions may get user's account flagged for selfbotting or modding and banned for doing so.
+Discord internally has mostly everything you'll need to retrieve data without directly calling the API or connecting to
+the Gateway. Performing such actions may get user's account flagged for selfbotting or modding and banned for doing so.
 
-## 11. Powercord Developers will never directly push in your plugin's repo
+### 11. Limit NSFW plugins
 
-Even if we have full access by being administrators of the organization, we'll never push to a plugin's repo unless
-we're invited to do so. We'll always prefer forking and making a PR.
+We're not against plugins providing NSFW features, but they should be properly labeled and handled. If your plugin can
```

Comment:
are apple bottom jeans and boots with the fur nsfw?

Reply:
😩